cmd/web3/main.go

```diff
@@ -215,6 +226,68 @@ func main() {
 				GetID(ctx, network.URL)
 			},
 		},
+		{
+			Name:  "start",
+			Usage: "Start a local GoChain development node",
+			Flags: []cli.Flag{
+				cli.BoolTFlag{
+					Name:  "detach, d",
+					Usage: "Run container in background.",
+				},
+				cli.StringFlag{
+					Name:  "env-file",
+					Usage: "Path to custom configuration file.",
+				},
+				cli.StringFlag{
+					Name:   "private-key,pk",
+					Usage:  "Private key",
+					EnvVar: pkVarName,
+				},
+			},
+			Action: start,
+		},
+		{
+			Name:  "myaddress",
+			Usage: fmt.Sprintf("Returns the address associated with %v", pkVarName),
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:   "private-key,pk",
+					Usage:  "Private key",
+					EnvVar: pkVarName,
+				},
+			},
+			Action: func(c *cli.Context) {
+				pk := c.String("private-key")
+				if pk == "" {
+					fmt.Printf("%v not set", pkVarName)
+					return
+				}
+				acc, err := web3.ParsePrivateKey(pk)
+				if err != nil {
+					log.Fatalln(err)
+				}
+				fmt.Print(acc.PublicKey())
+			},
```

Comment:
What if this was under `account` (no subcommand) instead?
`web3 account` and `web3 account create`

Reply:
Hmm, would there every be more to account than just the raw address?  That said, maybe `web3 account address` or something would be appropriate.